### DISCUSSION TITLE
how to print f and M as nc files?

### DISCUSSION THREAD

**Discussion by mullerrobert799** [Deltares/Wflow.jl#287] [Q&A] [answered]
I am wondering how could we print different parameters as *.nc files, for instance f, f_ and M? could you please guide us?

**Comment by mullerrobert799** [comment]
Dear Sir/Madam,

I am using the parameters _f_ and _M_ together with _KsatHorFrac_ while calibrating the model and would like to confirm their values to be used by the wflow. As input files (*.nc) show their values (estimated by HydroMT) but as I'm changing them directly in *.toml file during calibration and not sure either wflow has used the right ones, therefore, could you advise me on: how to get the print of them (_f_, _M_, _KsatVer_, horizontal saturated conductivity and other parameters, etc) after wflow simulation is completed?

Thanks and regards,

**Comment by verseve** [comment, marked as answer]
Note that the `M` parameter is not used by Wflow since v0.2.0, it reads the `f` parameter directly from the netCDF input file.

You can add the parameters to the TOML file, for example:

```
[output]
path = "output.nc"
compressionlevel = 1 

[output.vertical]   
f = "f"
kv₀ = "kv_0"

[output.lateral.subsurface]
kh₀ = "kh_0"
```

see also: https://deltares.github.io/Wflow.jl/dev/user_guide/step2_settings_file/#Grid-data
